packages/desktop-client/src/components/modals/ManageRulesModal.js

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from 'react';
+
+import { Modal } from 'loot-design/src/components/common';
+
+import ManageRules from '../ManageRules';
+
+export default function ManageRulesModal({ modalProps, payeeId }) {
+  let [loading, setLoading] = useState(true);
+  if (process.env.NODE_ENV === 'development') {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏would you mind using `isDevelopmentEnvironment()` util function here?